requirements.txt

```diff
@@ -1,9 +1,9 @@
 ninja
 cmake
 flaky
-numpy
-pennylane>=0.25
-pennylane-lightning>=0.25
+numpy<1.24
+pennylane<=0.28
+pennylane-lightning<=0.28
```

Comment:
Any update on this yet?

Reply:
not yet, focusing on this PR and vjp up to now.

Reply:
It works fine now.